tests/generic_config_updater/test_bgpl.py

```diff
@@ -14,6 +14,19 @@
 
 logger = logging.getLogger(__name__)
 
+def get_bgp_listener_config(duthost):
+    """ Get bgp listener config
+    """
+    cmds = "show ip bgp summary"
+    output = duthost.shell(cmds)
+    pytest_assert(not output['rc'],
+        "'{}' failed with rc={}".format(cmds, output['rc'])
+    )
+
+    bgp_listener_pattern = r".*BGPMonitor.*"
```

Comment:
Could you add one sample line as code comment? Suggest to make regex more strict.

Reply:
Talking about strict regex, from you sample line, you may use `  BGPMonitor$`

Reply:
Sample of stdout: `neighbor 11.0.0.1 description BGPMonitor\n neighbor 10.0.0.57 remote-as 64600\n neighbor 10.0.0.57 peer-group PEER_V4\n`
Ending character `$` will not match.
I change to runningconfig verification. Because some counter such as `MsgRcvd` will not be the same after rollback.